napkon_string_matching/terminology/loinc_search.py

```diff
@@ -12,9 +14,56 @@
 RESPONSE_NO_ENTRIES = "Keine passenden Einträge gefunden"
 RESPONSE_LOGIN = "Log In ‹ LOINC — WordPress"
 
+STOP_WORDS = set(stopwords.words('german'))
+
+EXAMPLEOUTCOME_ITEMS = [
+    """Dialyse:""",
+    """Invasiver Beatmung:""",
+    """Sauerstoff:""",
+    """Trachealkanüle:""",
+    """Respiratorisches Outcome (Sauerstoff-Therapie):"""]
+
+EXAMPLE_QUESTIONS = [
+    """Wurde der Barthel-Index vor der Entlassung durchgeführt?""",
+    """Fatigue Screening durchgeführt? Datum der Durchführung""",
+    """Wurde der "PROMIS Dyspnoe-Funktionseinschränkungen" durchgeführt?"""]
+
 logger = logging.getLogger(__name__)
 
 
+def make_alphanumeric(token: str):
+    """
+    removes all non alphanumerics in token
+            Parameters:
+                    token (str): The input char sequence
+            Returns:
+                    token (str): Same token without special chars
+    """
+    return ''.join(e for e in token if e.isalnum())
+
+
+def remove_stop_words(tokens: list = []):
+    """
+        removes stop words from tokenized list
+                Parameters:
+                        tokens (list): List of tokens
+                Returns:
+                        tokens (list): List of tokens without stop words
+    """
+    return [t for t in tokens if t.lower() not in STOP_WORDS]
+
+
+def get_search_syntax_string(tokens: list = []):
+    """
+    adds syntax for LOINC search and gets search string
```

Comment:
Would be nice to explain what this conversion does. It looks like it combines the tokens with an OR but also adds a ~ after each term. Can you document what the behaviour here is?